envoy/config/filter/http/jwt_authn/v2/config.proto

```diff
@@ -0,0 +1,216 @@
+
+syntax = "proto3";
+
+package envoy.config.filter.http.jwt_authn.v2;
```

Comment:
No "_" in package name please.

Reply:
Done

Reply:
@wora this is necessary if the path has underscore for consistency and some Go stuff.